coops-ui/tests/unit/AnnualReport.spec.ts

```diff
@@ -691,8 +690,26 @@ describe('AnnualReport - Part 3 - Submitting', () => {
 
     // stub address data
     vm.addresses = {
-      deliveryAddress: {},
-      mailingAddress: {}
+      'registeredOffice': {
+        'deliveryAddress': {
+          'streetAddress': 'delivery street address',
```

Comment:
Do the actual data properties need to exist for these tests? I mean, the tests worked before with only the stub addresses right?

Also below.

Reply:
They did indeed, they don't hurt the quality of the test but they certainly aren't necessary. I will clean that up!